src/static/js/modules/form-validation.js

```diff
@@ -6,34 +6,53 @@
 'use strict';
 
 var $ = require( 'jquery' );
+require( './jquery/cf_formValidator' ).init();
+require( './jquery/cf_notifier' ).init();
 
 function init() {
-  $( '.js-validate_form-not-empty' ).each( function() {
-    var $form = $( this );
-
-    $form.on( 'submit', function() {
-      var formIsEmpty = true;
-      var event;
-
-      $.each( $form.serializeArray(), function( index, element ) {
-        if ( element.value !== '' ) {
-          formIsEmpty = false;
-        }
-      } );
-
-      if ( formIsEmpty ) {
-        event = 'form:validate:empty';
-      } else {
-        event = 'form:validate:not_empty';
+  function sendNotification( elem, field ) {
+    var label = field.label;
+    var type;
+    var message;
+
+    for ( var prop in field.status ) {
+      if ( field.status[prop] === false ) {
```

Comment:
Some linters would want a `hasOwnProperty` call within for…in loops to avoid unexpected values bubbling up from the prototype (see top answer on http://stackoverflow.com/questions/684672/loop-through-javascript-object).

If you're always controlling what `field` will be here that may not be necessary. Just check that eslint doesn't complain about this line.